Kudu.Contracts/Settings/SettingsKeys.cs

```diff
@@ -45,5 +45,6 @@ public static class SettingsKeys
         public const string DoBuildDuringDeployment = "SCM_DO_BUILD_DURING_DEPLOYMENT";
         public const string RunFromZipOld = "WEBSITE_RUN_FROM_ZIP";  // Old name, will eventually go away
         public const string RunFromZip = "WEBSITE_RUN_FROM_PACKAGE";
+        public const string PurgeZipValue = "SCM_PURGE_ZIP_VAL";
```

Comment:
`SCM_PURGE_ZIP_VAL` does not convey well the meaning. I would call it `SCM_MAX_ZIP_PACKAGE_COUNT`.